docs/conf.py

```diff
@@ -67,17 +59,18 @@
 
 # General information about the project.
 project = 'tedana'
-copyright = '2017-2018, Elizabeth DuPre'
-author = 'Elizabeth DuPre'
+copyright = '2017-2018, tedana developers'
+author = 'tedana developers'
```

Comment:
Do we have a list of tedana developers somewhere? Would you like me to make an issue to discuss where this should go, and what the criteria for inclusion should be?

Reply:
That would be great ! Yes, it was listed as just me, and I think that's not quite right, thankfully 😄